docs/concepts/consensus-protocol/unl.md

```diff
@@ -0,0 +1,36 @@
+
+A recommended list can be identified by the public key of its publisher. Typically, recommended lists are associated with a website where they can be downloaded, but lists can also be relayed through the peer-to-peer network in case there are problems accessing the website.
+
+Currently, the default configuration for XRP Ledger servers uses two lists: one published by the XRP Ledger Foundation, and one published by Ripple. Typically, these lists are very similar to one another or even identical. The term _default UNL_ (sometimes abbreviated dUNL) refers to the set of validators included in these lists.
+
+Anyone can publish a signed list of validators in the correct format, which is a JSON document containing signed binary data. For more information on the format of a recommended list, see [Validator List Method](/docs/references/http-websocket-apis/peer-port-methods/validator-list/).
+
+Recommended validator lists need to be updated over time, to add more quality validators and remove validators that aren't as reliable or are retiring. Typically, a recommended validator list has an expiration time, because the publisher expects to have made an update available by that time. Lists also have a sequence number, such that the highest sequence number is the newest version of that list and supercedes any older versions. Lists can also have an activation date so that servers can coordinate when to switch to the new version, and the updated list has time to propagate to all servers that use it.
+
+Publishers aren't involved in day-to-day validation of new transactions, but they do wield significant power in selecting which validators are widely trusted. Server operators should be cautious in selecting which validator list publishers they trust, because carelessness on the part of the publishers could affect the reliability of the servers using those lists.
+
+## How to Get Your Validator on a Recommended List
+
+Each publisher can define their own criteria for being listed. However, the criteria for having your validator added to the default UNL typically include the following:
+
+- Run a validator with high uptime for at least a year and high agreement with the rest of the network.
+- Set up [domain verification](/docs/references/xrp-ledger-toml/#domain-verification) for your validator.
+- Be a separate and recognizeable entity in the XRP Ledger community—not, for example, an employee of a company that already runs a validator.
+- Run the validator in a separate physical location than most other validators. (So that, for example, an outage at a single data center does not cause many validators to go down.)
+
+Validator list operators may interview candidates for inclusion in the recommended list to confirm that they meet these and other requirements and that they have a commitment to continuing to run the server in the future.
```

Comment:
Do we have a link to a form or something where a validator can apply to get on the list?

Reply:
No, I think there might've been a form like that in the earlier days but not anymore. I believe the general policy is, "Don't reach out to us, we'll reach out to you." Validator performance and domain verification are public, so it's likely that the publishers will notice any validator that meets the requirements.

Reply:
Consensus does not involve 'throwing out' votes.

A Validator only listens to validators on its UNL and can only consider those votes. It can't consider or 'throw out' votes it doesn't know of, and that is from every other validator not on its UNL.